rpc/server/ipc_unix.go

```diff
@@ -28,10 +29,17 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
+const (
+	// On Linux, sun_path is 108 bytes in size
+	// see http://man7.org/linux/man-pages/man7/unix.7.html
+	maxPathSize = int(108)
+)
+
 // ipcListen will create a Unix socket on the given endpoint.
 func ipcListen(endpoint string) (net.Listener, error) {
-	if len(endpoint) > int(max_path_size) {
-		log.Warn(fmt.Sprintf("The ipc endpoint is longer than %d characters. ", max_path_size),
+	// account for null-terminator too
+	if len(endpoint)+1 > maxPathSize {
```

Comment:
Why did you replace it with len(endpoint)+1? The value of maxPathSize = int(108) stayed the same. Also why int(108) and not just 108?

Reply:
Check the comment above this line